network/endpoint_windows.go

```diff
@@ -14,16 +14,58 @@ import (
 	"github.com/Microsoft/hcsshim"
 )
 
+// ConstructEpName constructs endpoint name from netNsPath.
+func ConstructEpName(containerID string, netNsPath string, ifName string) string {
+	epName := ""
+	if netNsPath != "" {
+		splits := strings.Split(netNsPath, ":")
+		if len(splits) == 2 {
+			epName = splits[1]
```

Comment:
add comment here..briefly explaining two cases if and else?

Reply:
done